doc/nrf/gs_installing.rst

```diff
@@ -374,6 +374,12 @@ A toolchain provides a compiler, assembler, linker, and other programs required
 
 .. rst-class:: numbered-step
 
+Set the following environment variables to ensure the Zephyr SDK is detected
```

Comment:
I worry that this might be missed since it is outside the Ubuntu/macOS/Windows tab system

Getting it inside would require modifying `develop/getting_started/index.rst`, but that is a Zephyr rst file

In fact, your related PR https://github.com/zephyrproject-rtos/zephyr/pull/46319 modifies this file to include a reference to this issue directly (without specifying that the correct value is ``zephyr``), So when that commit gets upmerged, we'll have it mentioned twice in a row (since in that PR, the notice is also outside the OS tabs). 

Not sure how to proceed here. Without changes to the Zephyr docs or significant changes to our own docs, what you've written is probably the best that can be done. But it may need to be removed on the next Zephyr upmerge.

Reply:
Please check https://github.com/zephyrproject-rtos/zephyr/pull/46365

Reply:
Actually, let me see if my commit below can fix the formatting without having to edit the Zephyr part, given the concerns about my edits there.